runtime/bindings/python/src/pyopenvino/core/ie_infer_queue.cpp

```diff
@@ -145,89 +140,89 @@ class InferQueue {
-        return self._user_ids;
-    });
-}
+// void regclass_InferQueue(py::module m) {
+//     py::class_<InferQueue, std::shared_ptr<InferQueue>> cls(m, "InferQueue");
+
+//     cls.def(py::init([](InferenceEngine::ExecutableNetwork& net, size_t jobs) {
+//                 if (jobs == 0) {
+//                     const InferenceEngine::ExecutableNetwork& _net = net;
+//                     jobs = (size_t)Common::get_optimal_number_of_requests(_net);
+//                 }
+
+//                 std::vector<InferRequestWrapper> requests;
+//                 std::queue<size_t> idle_handles;
+//                 std::vector<py::object> user_ids(jobs);
+
+//                 for (size_t handle = 0; handle < jobs; handle++) {
+//                     auto request = InferRequestWrapper(net.CreateInferRequest());
+//                     // Get Inputs and Outputs info from executable network
+//                     request._inputsInfo = net.GetInputsInfo();
+//                     request._outputsInfo = net.GetOutputsInfo();
+
+//                     requests.push_back(request);
+//                     idle_handles.push(handle);
+//                 }
+
+//                 return new InferQueue(requests, idle_handles, user_ids);
+//             }),
+//             py::arg("network"),
+//             py::arg("jobs") = 0);
+
+//     cls.def(
+//         "_async_infer",
+//         [](InferQueue& self, const py::dict inputs, py::object userdata) {
+//             // getIdleRequestId function has an intention to block InferQueue
+//             // until there is at least one idle (free to use) InferRequest
+//             auto handle = self.getIdleRequestId();
+//             // Set new inputs label/id from user
+//             self._user_ids[handle] = userdata;
+//             // Update inputs of picked InferRequest
+//             if (!inputs.empty()) {
+//                 Common::set_request_blobs(self._requests[handle]._request, inputs);
+//             }
+//             // Now GIL can be released - we are NOT working with Python objects in this block
+//             {
+//                 py::gil_scoped_release release;
+//                 self._requests[handle]._start_time = Time::now();
+//                 // Start InferRequest in asynchronus mode
+//                 self._requests[handle]._request.start_async();
+//             }
+//         },
+//         py::arg("inputs"),
+//         py::arg("userdata"));
+
+//     cls.def("is_ready", [](InferQueue& self) {
+//         return self._is_ready();
+//     });
+
+//     cls.def("wait_all", [](InferQueue& self) {
+//         return self.waitAll();
+//     });
+
+//     cls.def("get_idle_request_info", [](InferQueue& self) {
+//         return self._getIdleRequestInfo();
+//     });
+
+//     cls.def("set_infer_callback", [](InferQueue& self, py::function f_callback) {
+//         self.setCustomCallbacks(f_callback);
+//     });
+
+//     cls.def("__len__", [](InferQueue& self) {
```

Comment:
Maybe we can remove this commented code

Reply:
We'll adapt this code in the next PR, so decided to temporarily comment it